helm/cluster-vsphere/values.yaml

```diff
@@ -1,9 +1,9 @@
 apiServer:
   enableAdmissionPlugins: "NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota,PersistentVolumeClaimResize,DefaultStorageClass,Priority,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook"
-  featureGates: "TTLAfterFinished=true"
+  featureGates: ""
```

Comment:
As far as I see, it is enabled by default in newer k8s versions https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates-removed/ so it is safe to delete it. nice.